src/utils/get-testcafe-version.ts

```diff
@@ -1,3 +1,19 @@
-export default function (): string {
-    return require('../../package.json').version;
+import logEntry from './log-entry';
+import debug from 'debug';
+
+const LOGGER = debug('testcafe:version');
+
+export function logTestCafeVersion (): void {
+    logEntry(LOGGER, getTestCafeVersion());
+}
```

Comment:
This function uses only 1 time. So, it's better to use in the `src/index.ts` file a plain instruction instead of function call.
```js
import testcafeVersionLogger from ....
import getTestCafeVersion from ...

testcafeVersionLogger(getTestCafeVersion());
```